dev/fattest.simplicity/src/componenttest/rules/repeater/FeatureReplacementAction.java

```diff
@@ -60,6 +60,7 @@
         featureNameMapping.put("ejbTest", "enterpriseBeansTest");
         featureNameMapping.put("javaee", "jakartaee");
         featureNameMapping.put("javaeeClient", "jakartaeeClient");
+        featureNameMapping.put("javaMail", "jakartaMail");
```

Comment:
I think you want this to be mail not jakartaMail

Reply:
Made the change.